llvm/test/CodeGen/PowerPC/fp-classify.ll

```diff
@@ -182,7 +181,8 @@ define <4 x i1> @abs_isinfv4f32(<4 x float> %x) {
 ; P9-NEXT:    xvabssp 0, 34
 ; P9-NEXT:    addi 3, 3, .LCPI6_0@toc@l
 ; P9-NEXT:    lxv 1, 0(3)
-; P9-NEXT:    xvcmpeqsp 34, 0, 1
+; P9-NEXT:    xvcmpeqsp 0, 0, 1
```

Comment:
`isCondCodeLegalOrCustom` not working well here? PPC has `xvcmpeqsp` but no `xvcmpnesp`.